cfme/automate/explorer.py

```diff
@@ -296,12 +296,15 @@ def is_enabled(self):
 
     @classproperty
     def default(cls):
-        if not hasattr(cls, "_default_domain"):
-            cls._default_domain = version.pick({
-                version.LOWEST: None,
-                '5.3': cls('Default')
-            })
-        return cls._default_domain
+        if on_rtd:
+            return "Collapsed at runtime"
```

Comment:
maybe this one should return `cls('Default')` as does the line 305? Otherwise I am fine ith merging this.